### PR TITLE
Merge jaeger-1.0 into mpOpenTracing-1.3

### PR DESCRIPTION
**Pull request by fmhwong**
It was decided having `jaeger-1.0` might be confusing to customers as the feature does not include Jaeger client jars.  Just to merge `jaeger-1.0` into `mpOpenTracing-1.3`.

Fixes #9271

### CODE REVIEW

**Comment by fmhwong** [comment]
#build

**Comment by fmhwong** [comment]
#build

**Comment by fmhwong** [comment]
@tevans78 Jaeger will be enabled when it is beta or `JAEGER_ENABLEMENT=true` is defined as a system property or environment variable.

File: dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java [hunk 1]
```diff
@@ -327,4 +338,28 @@ private static String resolveValue(String value) {
...
+            }
+        } catch (Exception e) {
+            Tr.debug(tc, "Exception getting InstalledProductInfo: " + e.getMessage());
+            e.printStackTrace();
+        }
+        return false;
+    }
+    
+    private static boolean isEnabledByUser() {
+        Boolean isEnabled  = getBooleanProperty(ENV_JAEGER_ENABLEMENT);
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java]: Why do you need this environment variable?

Comment (fmhwong) [reply dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java]: This is used to enable Jaeger in OL builds as OL doesn't have beta builds.

Comment (tevans78) [reply dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java]: No but it does have development builds ... https://openliberty.io/downloads/#development_builds

Those contain all features, not just the GA ones. What does ProductInfo.getEdition() return for those?

Comment (fmhwong) [reply dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java]: Development builds have same properties as a GA build.  These are what I got from one.

```
com.ibm.websphere.productId=io.openliberty
com.ibm.websphere.productOwner=Open Liberty Project
com.ibm.websphere.productVersion=19.0.0.11
com.ibm.websphere.productName=Open Liberty
com.ibm.websphere.productInstallType=Archive
com.ibm.websphere.productEdition=Open
com.ibm.websphere.productLicenseType=EPL
```

Comment (tevans78) [reply dev/com.ibm.ws.microprofile.opentracing.jaeger/src/com/ibm/ws/microprofile/opentracing/jaeger/JaegerTracerFactory.java]: OK, fair enough

**Comment by fmhwong** [comment]
The one failure has been reported in 267449 and it is not related to OpenTracing and Jaeger.

**Pull request merged**